app/pb_kits/playbook/pb_label_pill/_label_pill.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname("pb_label_pill_kit")) do %>
-  <%= object.label %>
-  <%= object.pill %>
+    class: object.classname) do %>
+      <%= pb_rails("caption", props: { text: object.label }) %>
```

Comment:
This was getting passed `classname: "pb_label_pill_label"`

Reply:
got both class names restored to their former glory. Spec still passes as is.